manifests/cluster-rbac/application-controller/argocd-application-controller-clusterrolebinding.yaml

```diff
@@ -13,4 +13,3 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: argocd-application-controller
-  namespace: argocd
```

Comment:
Does `kubectl apply` automatically populate this field with the value from `-n`?

Reply:
We may just have to direct folks to use kustomize if they want a non default namespace

Reply:
I suppose it may make more sense to just document that users need to run `kustomize edit set namespace <name>` for themselves if installing cluster-scoped argo-cd in a different namespace?

Reply:
Ah, looks like this is documented already 
https://argo-cd.readthedocs.io/en/stable/getting_started/#1-install-argo-cd

I will update the issue pointing to this and close this PR then

Thanks @crenshaw-dev